collections/soldiers.js

```diff
@@ -0,0 +1,34 @@
+const MongoClient = require('mongodb').MongoClient;
+
+const url = 'mongodb://localhost:27017';
+const dbName = 'myproject';
+
+function checkData(soldierData) {
+  if (/tt[0-9]{7}/.test(soldierData["id"]) &&
+    soldierData["name"] &&
+    soldierData["rank"] &&
+    soldierData["limitations"] &&
+    Object.keys(soldierData).length === 5) {
+    return true;
```

Comment:
instead of
`if (...) { return true } else { return false }`
just
`return (...)`

We're writing in javascript, the land where types are irrelevant and shortness is holy ;)
(I wish I was joking, but it's sort of true, and we'll respect that 🌝  )